### PR TITLE
feat: generate social sms and email connector guides

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Add connector guides to the new "Integrations" page. The guides are generated by pulling remote README.md files from remote GitHub connector repos, using `docusaurus-plugin-remote-content` plugin.

The script is located at `docs/integrations/generate.js`.

image

### CODE REVIEW

File: docs/integrations/generate.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    description:
+      'The SMTP is an internet standard communication protocol for electronic mail transmission.',
+    category: 'email',
+  },
+  {
+    id: 'twilio-sms',
+    name: 'Twilio SMS',
+    description: 'Twilio provides programmable communication tools for phone calls and messages.',
+    category: 'sms',
+    logoFilename: 'twilio.svg',
```
Comment (gao-sun) [inline docs/integrations/generate.js:179]: is it possible to read a file for these metadata props?

File: docs/integrations/generate.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+ *
+ * @type Array<{ id: string; name: string; description: string; category: 'social' | 'sms' | 'email'; logoFilename?: string; }>
+ */
+const connectors = [
+  {
+    id: 'google',
+    name: 'Google',
+    description: 'Google is a principal search engine technology and email service provider.',
+    category: 'social',
+  },
```
Comment (gao-sun) [inline docs/integrations/generate.js:32]: can we read the connector list from the remote repo as well?

Comment (charIeszhao) [reply docs/integrations/generate.js:32]: It's kinda hard to do it, part of the reason being that there are several connectors that we do not want to show on the "Integrations" page, such as those mock connectors. Therefore, even if we read connectors from the remote repo, we still have to maintain a filter list to filter out those ones. Also, we have to maintain the mapping of the connectors and the "categories" (social / sms / email)...

Given that, I don't think it worths the effort to do it.

File: docs/integrations/generate.js (hunk 1 above)
Comment (charIeszhao) [reply docs/integrations/generate.js:179]: It is possible, but let's talk about it offline to see if it is necessary.

File: sidebars.js [hunk 3]
```diff
@@ -73,18 +73,34 @@ const sidebars = {
...
+    // },
+    // { type: 'autogenerated', dirName: 'integrations/sso' },
+    // {
+    //   type: 'link',
+    //   label: 'Third-party apps',
+    //   customProps: { additionalLabel: '(Logto as Idp)' },
+    //   href: '#',
+    //   className: 'sidebar-section',
+    // },
+    // { type: 'autogenerated', dirName: 'integrations/third-party' },
```
Comment (gao-sun) [inline sidebars.js:103]: why is this commented?

Comment (charIeszhao) [reply sidebars.js:103]: The SSO and 3rd-party app guides are available in the follow-up PR #668.

File: docs/integrations/generate.js (hunk 1 above)
Comment (charIeszhao) [reply docs/integrations/generate.js:179]: Discussed offline, and created a task to improve the script. https://linear.app/silverhand/issue/LOG-8950/improve-the-pull-connector-readme-script

File: docs/integrations/generate.js (hunk 2 above)
Comment (charIeszhao) [reply docs/integrations/generate.js:32]: https://linear.app/silverhand/issue/LOG-8950/improve-the-pull-connector-readme-script

**Pull request merged**